engine/runtime/src/test/java/org/enso/compiler/ErrorCompilerTest.java

```diff
@@ -27,6 +29,16 @@ public void unfinishedLiteral1() throws Exception {
     assertSingleSyntaxError(ir, IR$Error$Syntax$UnclosedTextLiteral$.MODULE$, "Unclosed text literal", 6, 28);
   }
 
+  @Test
+  public void dotUnderscore() throws Exception {
+    var ir = parse("""
+    run op =
+      op._
```

Comment:
What happens if you write `op._.op`?

Reply:
That will yield the same error. `_` is only allowed at 1st segment, I think.

Reply:
Let's add a test for this case as well

Reply:
As you wish: 484c6d9